tests/zero_bounce_test_case.py

```diff
@@ -246,3 +246,61 @@ def test_delete_file_valid(self):
         self.assertEqual(response.message, "File Deleted")
         self.assertEqual(response.file_id, "5e87c21f-45b2-4803-8daf-307f29fa7340")
         self.assertEqual(response.file_name, "emails.txt")
+
+    def test_find_email_status_invalid(self):
+        self.requests_mock.get.return_value = mock_response = MockResponse({
+            "email": "",
+            "domain": "example.com",
+            "format": "unknown",
+            "status": "invalid",
+            "sub_status": "no_dns_entries",
+            "confidence": "undetermined",
+            "did_you_mean": "",
+            "failure_reason": "",
+            "other_domain_formats": []
+        })
+
+        response = self.zero_bounce_client.find_email(
+            "example.com", "John", "", "Doe"
```

Comment:
if you make this request you will actually receive a different response
![image](https://github.com/zerobounce/zero-bounce-python-sdk-setup/assets/39882567/b6cd76ff-03b8-49da-86fc-fdadc997e2c7)
i think that for this invalid example you can send only the domain so that the mocked response would be accurate

Reply:
Sometimes I receive "other domains formats" int the response, sometimes I don't. I wanted a test case in which those are also received in order to ensure they are parsing well. So yeah, my goal was not accuracy of the request, quite the contrary: wanted some mock parameters for a specific mock response.

Reply:
I don't think the api will return `other_domain_formats` for an invalid response. There is the valid test case witch handles `other_domain_formats`.